packages/query/src/fragments/seo/template/open-graph-metadata.graphql

```diff
@@ -0,0 +1,34 @@
+fragment OpenGraphMetadataFrag on RenderedTemplate {
+	uri
+	connectedNode {
+		__typename
+		... on NodeWithTitle {
+			title
+		}
+		... on NodeWithExcerpt {
+			excerpt
+		}
+		... on NodeWithFeaturedImage {
+			featuredImage {
+				node {
+					sourceUrl
+					mediaDetails {
```

Comment:
```suggestion
					id
					sourceUrl
					mediaDetails {
```

Reply:
We are not using apollo caching. No need for 'id'